eng/Baseline.Designer.props

```diff
@@ -31,19 +31,6 @@
     <BaselinePackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="[2.2.0, )" />
     <BaselinePackageReference Include="Microsoft.Extensions.ObjectPool" Version="[2.2.0, )" />
   </ItemGroup>
-  <!-- Package: Microsoft.AspNetCore.ApiAuthorization.IdentityServer-->
-  <PropertyGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.ApiAuthorization.IdentityServer' ">
-    <BaselinePackageVersion>2.2.0-preview-35687</BaselinePackageVersion>
-  </PropertyGroup>
-  <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.ApiAuthorization.IdentityServer' AND '$(TargetFramework)' == 'netstandard2.0' ">
-    <BaselinePackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="[2.2.0, )" />
-    <BaselinePackageReference Include="Microsoft.AspNetCore.Identity.UI" Version="[2.2.0, )" />
-    <BaselinePackageReference Include="IdentityServer4" Version="[2.3.0-preview1-update2, )" />
-    <BaselinePackageReference Include="IdentityServer4.AspNetIdentity" Version="[2.3.0-preview1-update2, )" />
-    <BaselinePackageReference Include="IdentityServer4.EntityFramework" Version="[2.3.0-preview1-update1, )" />
-    <BaselinePackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="[2.2.0, )" />
-    <BaselinePackageReference Include="Microsoft.AspNetCore.Mvc" Version="[2.2.0, )" />
-  </ItemGroup>
```

Comment:
Revert this change.

Reply:
Same comment as below.

I can revert these two, but it doesn't matter.

Reply:
Thanks for doing this. It matters because the 'release/3.0' branch does not control the baseline files and changing them here will cause merge conflicts.